extension/src/popup/components/identicons/KeyIdenticon/index.tsx

```diff
@@ -27,6 +32,12 @@ export const KeyIdenticon = ({
           "--Icon-dimension": "1.5rem",
         }
       : {}),
+    ...(customSize
+      ? {
+          "--Icon-padding": customSize.padding,
+          "--Icon-dimension": customSize.dimension,
+        }
```

Comment:
this is not great - but we have a one-off version of the Identicon in transaction details.

We can circle back later to see if we have any other sizes and come up with a better API for this